tests/test_select.py

```diff
@@ -40,3 +41,17 @@ def test_select_multiple_residues(selection):
     vertices_count = len(bob.evaluate().data.vertices)
     assert vertices_count == len(selection[1])
     assert (bob.named_attribute("res_id", evaluate=True) == selection[1]).all()
+
+
+def test_gfp_selection():
+    structure = mn.Molecule.fetch("1EMA")
+
+    # select AA and create a named selection
+    structure.select.is_amino_acid().store_selection("Protein")
+    structure.add_style("cartoon", selection="Protein")
+    assert structure.bob.named_attribute("Protein").sum() == 1771
```

Comment:
you can just call `structure.named_attribute(` rather than explicitly having to do `bob` first, as `bob` is actually a mix-in for the `MolecularEntity`.

Reply:
Addressed.